### DISCUSSION TITLE
!.31 and 1.32 releases

### DISCUSSION THREAD

**Discussion by rawsome1** [Joshua-Riek/ubuntu-rockchip#550] [General]
Releases 1.31 and 1.32 on my OPI5+, the Bluetooth is erratic to non working and the WIFI does not show up at all. I'm using 1.29 right now and Bluetooth and WIFI work. I'm running the OS from the EMMC module. Anyone else experiencing this? BTW, I also have the OPI5B and have no problems in these areas.

**Comment by JFLim1** [comment]
> Releases 1.31 and 1.32 on my OPI5+, the Bluetooth is erratic to non working and the WIFI does not show up at all. I'm using 1.29 right now and Bluetooth and WIFI work. I'm running the OS from the EMMC module. Anyone else experiencing this? BTW, I also have the OPI5B and have no problems in these areas.

Do encounter wifi or bluetooth not enabled after boot up sometimes.  Started with image v1.29 and upgraded to latest updates/upgrades.

Had to re-enable wifi if wifi is not enabled after boot up using the following:
sudo modprobe -r 8852be
sudo modprobe -i 8852be

**Comment by rawsome1** [comment]
Tried those, but it had no affect. WIFI does not show up in settings.

**Reply by JFLim1** [reply]
At time the above commands also does not work on my current image and reboot usually restore the wifi and if it doesn't, run the above two commands again and wifi is enabled.  That is the experience so far.

**Comment by Joshua-Riek** [comment]
There is no need to re install your system every release. I tested this last night and wifi and Bluetooth works perfectly, what card are you using? Is the antennas attached fully?

**Comment by rawsome1** [comment]
Well the WIFI and Bluetooth work with 1.29 release, so I know the antennas are connected. I didn't know I can upgrade without reinstalling the system. Are there instructions for that?
